app/src/main/res/layout/activity_detail.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This resource is unused and is basically the same as "details_fragment". Why is that? What do you think of leaving unused resources in the project?

Reply:
I've always preferred to use fragments instead of activities, I feel I have more freedom in defining them, not to mention that I could migrate to Navigation easily, leaving only an activity.